### PR TITLE
added para_pat

### PR DESCRIPTION
**Pull request by bhavitvyamalik**
Dataset link : https://figshare.com/articles/ParaPat_The_Multi-Million_Sentences_Parallel_Corpus_of_Patents_Abstracts/12627632
Working on README.md currently

### CODE REVIEW

File: datasets/para_pat/para_pat.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    def _info(self):
+        if self.config.name in type1_datasets_features:
+            features = datasets.Features(
+                {
+                    "index": datasets.Value("int32"),
+                    "family_id": datasets.Value("int32"),
+                    "src_lang": datasets.Value("string"),
+                    "src_abs": datasets.Value("string"),
+                    "tgt_lang": datasets.Value("string"),
+                    "tgt_abs": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/para_pat/para_pat.py:156]: can you use the `Translation` feature type here ?
You can find an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py)

File: datasets/para_pat/para_pat.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                }
+            )
+        elif self.config.name in type2_datasets_features:
+            features = datasets.Features(
+                {
+                    "src_lang": datasets.features.Sequence(datasets.Value("string")),
+                    "tgt_lang": datasets.features.Sequence(datasets.Value("string")),
```
Comment (lhoestq) [inline datasets/para_pat/para_pat.py:163]: same here

**Review by lhoestq — COMMENTED** [review]
Really cool !

I left a comment about the feature type. Can you do this change please ?
After the change you'll need to regenerate the dataset_infos.json

Also don't forget to fill the tags of the dataset card ;)

**Comment by bhavitvyamalik** [comment]
Updated with Translation feature type. Working on dataset tags and README

File: datasets/para_pat/para_pat.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+        ),
+        ParaPatConfig(
+            language_pair=("ru", "uk"),
+            url="https://ndownloader.figshare.com/files/23855465",
+        ),
+        ParaPatConfig(
+            language_pair=("en", "pt"),
+            url="https://ndownloader.figshare.com/files/23855441",
+        ),
+    ]
```
Comment (lhoestq) [inline datasets/para_pat/para_pat.py:199]:
```suggestion
    ]
    BUILDER_CONFIG_CLASS = ParaPatConfig
```

**Review by lhoestq — APPROVED** [review]
Thanks !
Looks all good now

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**